### DISCUSSION TITLE
VS Code - Salesforce - Code Diff - Allow Direction Change

### DISCUSSION THREAD

**Discussion by aaron-rodriguez-j2** [forcedotcom/salesforcedx-vscode#5473] [Ideas]
**Is your feature request related to a problem? Please describe.**

The Diff plugin is functioning properly. However, by default, it shows the local file on the right, and the server file on the left. This doesn't make sense because deployment would occur from the local file to the server file, which should be left to right.

**Describe the solution you'd like**

Make the local file appear on the left, and the server file appear on the right.

**Describe alternatives you've considered**

Allow configuration of a setting that determines whether the local file is on the left or right.

**Additional context**

![image](https://user-images.githubusercontent.com/85955895/122091109-74ba6480-cdd6-11eb-905e-58a16cc6cd71.png)

**Comment by aaron-rodriguez-j2** [comment]
@smaddox-sf 

Is there anything I can do so that we implement this change? I think it is counter-intuitive for the local (source) file to be on the right whereas the server (target) file would be on the left. Most compare applications would have the source file on the left and the target file on the right.

**Comment by smaddox-sf** [comment]
Hi @aaron-rodriguez-j2 - Thanks for the feedback.  We researched other Diff viewers and most have the local/working copy on the right with the original on the left.  Whenever possible, we seek to make the experience for Salesforce developer similar to other industry experiences to reduce the number of things you need to learn.  We'll consider adding an option to change the view in the future.  We would also be open to a PR on this that adds a setting to give users an option to reverse the view.

**Comment by aaron-rodriguez-j2** [comment]
thanks smaddox-sf
testing line breaks1
testing line breaks2

**Comment by fprovot** [comment]
I agree, I have the same reaction, searching to invert the columns. The "ref" source should be on the right (or we should be able to switch).
